vllm/v1/core/sched/scheduler.py

```diff
@@ -188,6 +189,15 @@ def __init__(
         )
         self.use_pp = self.parallel_config.pipeline_parallel_size > 1
 
+        # Used to derive model's MFU stats for each SchedulerOutput
+        self.perf_metrics: ModelMetrics | None
+        try:
+            self.perf_metrics = ModelMetrics(vllm_config)
```

Comment:
We should only enable this if `log_stats` is `True`

Reply:
Done.